config/routes.rb

```diff
@@ -140,6 +140,7 @@
 
       # POST /api/vi/ping-thing/
       post 'ping-thing/:network', to: 'ping_things#create', as: 'ping_thing'
+      post 'ping-thing-batch/:network', to: 'ping_things#create_multiple', as: 'ping_thing_batch'
```

Comment:
```suggestion
      post 'ping-things/batch/:network', to: 'ping_things#create_multiple', as: 'ping_thing_batch'
```
I am still concerned about using singular paths for resources instead of plurals, it does not follow REST API conventions. Singular form would be ok if we have only one ping thing.

https://stackoverflow.com/a/21809963/3868595
https://stackoverflow.com/questions/6845772/should-i-use-singular-or-plural-name-convention-for-rest-resources#comment52501175_6846057
https://restfulapi.net/resource-naming/